package/yast2-registration.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Mon Mar 13 11:12:19 UTC 2017 - gsouzadossantos@suse.com
+
+- Remember the state of the checkbox when leaving the dialog 
+  (bsc#1026155) and correct the filter to always show installed
+  addons, even if they are beta versions.
+
```

Comment:
Just a minor one: please, add a line with the new version number like we do https://github.com/gilsonsouza/yast-registration/blob/169fdad67e062ebf7b5b3a99f174f20ce781f4e5/package/yast2-registration.changes#L13.